VideoLocker/res/values/strings.xml

```diff
@@ -107,6 +108,8 @@
     <string name="login_title">Sign in to edX</string>
     <!-- Accessible content description of close button on Sign in and Sign up screen -->
     <string name="close">Close</string>
+    <!-- Accessible content description of cancel button to cancel video download -->
+    <string name="cancel">Cancel</string>
```

Comment:
:heavy_check_mark: It should probably be before the alert dialog strings though.